magefile.go

```diff
@@ -65,3 +67,8 @@ func (Fmt) Lint() error {
 	lintCmd := "golangci-lint"
 	return sh.RunV(lintCmd, "run")
 }
+
+// enabling verbose as initial setup.
+func init() {
+	os.Setenv("MAGEFILE_VERBOSE", "true")
```

Comment:
This is shouldn't be here. One can easilly run mage with verbose enabled by providing `-v` flag to that.

```
mage -v
// or
docker-compose run --rm sdk mage -v test:run

```

Reply:
Thanks for info, All is done in separate commits.
I will create a new ticket for go-chassis and related projects for it.